src/main/python/rlbot/gui/preset_editors.py

```diff
@@ -94,7 +94,7 @@ def load_preset_cfg(self):
         file_path = QtWidgets.QFileDialog.getOpenFileName(self, 'Load Config', '', 'Config Files (*.cfg)')[0]
         if not os.path.isfile(file_path):
             return
-        if os.path.splitext(file_path)[1] != "cfg":
+        if os.path.splitext(file_path)[1] != ".cfg":
```

Comment:
we should use https://docs.python.org/3/library/pathlib.html


import pathlib

print(pathlib.Path('yourPathGoesHere').suffix)

Or this and remove the dot

import os.path
extension = os.path.splitext(filename)[1][1:]